### DISCUSSION TITLE
Spec Profiles not Functioning as Expected

### DISCUSSION THREAD

**Discussion by dfurray** [SFX-WoW/Masque#212] [Issues] [answered]
Not limited to but tested with a Death Knight. 
Enabled spec profiles, made 3 profiles Blood for Blood etc. Made blood red, frost blue and unholy green. 
Spec do save, but the color doesn't update on switching spec. 
Saving definitely works as if I change spec log out and back in the color does update to the right color.

**Comment by StormFX** [comment]
I'll look into it.

**Comment by dfurray** [comment]
Reopened because I thought it was fixed but it's doing the same thing I'm so confused.

**Comment by StormFX** [comment]
I just tested it and it's working fine on my end. Does it update properly when you manually change specs?

**Comment by dfurray** [comment]
I've tested it a lot, and it's switching but it's not saving all the colors which is weird, but it could be caused by the skins so I'm about to test them, but I'm not getting any errors which makes it harder to test. I do have lua errors turned on.

**Comment by dfurray** [comment, marked as answer]
So, I've deleted the saved variable file and started from scratch, it seems to be working now. 
I'm going to work towards getting everything how it was before and see if it breaks again. (I found debug mode so I can be more helpful if I can recreate the issue). 

Hopefully it's just a wtf moment. Sorry for wasting your time.

**Comment by StormFX** [comment]
Hope you figure it out.
